src/main/java/org/rumbledb/types/ItemTypes.java

```diff
@@ -18,7 +18,7 @@
  *
  */
 
-package sparksoniq.semantics.types;
+package org.rumbledb.types;
 
 public enum ItemTypes {
```

Comment:
I was initialy investigating the idea whether ItemTypes  enum and ItemType classes could be combined to generate a single source of truth. But from a code organization perspective the existing organization seems nice. 

I have an alternative suggestion however. The way things are, ItemTypes feels like an underlying type utility enum for ItemType class. Hence, wouldn't it be nice if we made the access modifier package private here? I believe developers should never access ItemTypes directly, and always go through the ItemType class. Do you think that makes a sound design or am I missing something?

Reply:
Exactly -- the masterplan is to completely hide ItemTypes from outside ItemType.

However, this is a lot of work to do because it is used at a lot of places, and this will be done later in a separate PR.